aiida_vasp/io/eigenval.py

```diff
@@ -4,32 +4,80 @@
 
 import numpy as np
 
-from .parser import BaseParser
+from aiida_vasp.utils.aiida_utils import get_data_class
+from aiida_vasp.io.parser import BaseFileParser
 
 
-class EigParser(BaseParser):
+class EigParser(BaseFileParser):
```

Comment:
Please add unit tests for this class in aiida_vasp/io/tests/test_eigenval_io.py

Reply:
Added a test for EigParser and a corresponding EIGENVAL file.